README.md

```diff
@@ -159,6 +161,9 @@ Run `docker-compose up`. This will start the following services:
   This worker type is not used by default by the API server, but this can be modified by uncommenting `- FROCKET_INVOKER=aws_lambda` in the `docker-compose.yml` file. 
   It is also called directly by unit tests.
 
+The empty local sub-directory `./data` on the host is mapped to Funnel Rocket's containers (the API Server and both worker types) at the same location: as `./data` under the container's `WORKDIR`.
+This allows you to register and query local datasets with the Docker-based setup
+
 To make jobs run faster, you can scale up the number of workers, e.g. `docker-compose up --scale frocket-queue-worker=4`. Workers only take about 50-60mb RAM each.
 
 #### Testing the Setup
```

Comment:
I think we should add a remark that the docker-compose needs to be up and running?

Reply:
please take a look at the updated README - now that tests are running via docker, is it unclear that testing happens right after launching services?

Reply:
I think so, since the testing section mentions how to start all these guy's up locally.
And I assume that the "correct" way to do this is to do `docker-compose up` and then test